### PR TITLE
Fix TMO error #1166

### PR DESCRIPTION
**Pull request by edgarMejia**

### CODE REVIEW

File: src/es/tumangaonline/src/eu/kanade/tachiyomi/extension/es/tumangaonline/TuMangaOnline.kt [hunk 1]
```diff
@@ -200,57 +201,92 @@ class TuMangaOnline : ParsedHttpSource() {
...
     }
 
-    override fun pageListParse(document: Document): List<Page> = mutableListOf<Page>().apply {
-        document.select("div#viewer-container > div.viewer-image-container > img.viewer-image")?.forEach {
-            add(Page(size, "", it.attr("src")))
+    override fun pageListParse(response: Response): List<Page> {
+        val pages = mutableListOf<Page>()
+        val body = response.body()!!.string()
+        var images = ""
+        val script = """
```
Comment (arkon) [inline src/es/tumangaonline/src/eu/kanade/tachiyomi/extension/es/tumangaonline/TuMangaOnline.kt]: Does this really need to be run as JS?

Comment (edgarMejia) [reply src/es/tumangaonline/src/eu/kanade/tachiyomi/extension/es/tumangaonline/TuMangaOnline.kt]: I'm not sure, it's my first time using Duktape, for now it's all I can do.

Comment (arkon) [reply src/es/tumangaonline/src/eu/kanade/tachiyomi/extension/es/tumangaonline/TuMangaOnline.kt]: I meant more along the lines of can't you just get this from the page source in Kotlin?

Comment (edgarMejia) [reply src/es/tumangaonline/src/eu/kanade/tachiyomi/extension/es/tumangaonline/TuMangaOnline.kt]: Now they use canvas, I don't know how to get the images without JS, I'm not sure if there's another way to do it with Kotlin.

Like this:

```html
<div class="viewer-image-container text-center">
    <figure>
        <canvas id="vYha5qsLE5" class="viewer-image" oncontextmenu="return false;" height="1821" width="1280"></canvas>
    </figure>
</div>
```

Comment (arkon) [reply src/es/tumangaonline/src/eu/kanade/tachiyomi/extension/es/tumangaonline/TuMangaOnline.kt]: That's actually script tag on the page that sets all of them, although I guess this is a bit more future-proof?

Comment (edgarMejia) [reply src/es/tumangaonline/src/eu/kanade/tachiyomi/extension/es/tumangaonline/TuMangaOnline.kt]: Hope so.

Also this solves the error that doesn't allow you to download many chapters.
Now you can download without limits.

**Pull request merged**